extensions/sql-migration/src/wizard/skuRecommendationPage.ts

```diff
@@ -86,6 +86,13 @@ export class SKURecommendationPage extends MigrationWizardPage {
 
 		this._detailsComponent = this.createDetailsComponent(view); // The details of what can be moved
 
+		const refreshAssessmentButton = this._view.modelBuilder.button().withProps({
+			label: constants.REFRESH_ASSESSMENT_BUTTON_LABEL,
+			width: 130
+		}).component();
+
+		refreshAssessmentButton.onDidClick(() => this.constructDetails());
+
```

Comment:
please add the disposable returned from the onDidClick method def to the dispososables list
this._disposables.push(
refreshAssessmentButton.onDidClick(() => this.constructDetails())
);